test/e2e/framework/node/init/init.go

```diff
@@ -30,14 +31,10 @@ import (
 func init() {
 	framework.NewFrameworkExtensions = append(framework.NewFrameworkExtensions,
 		func(f *framework.Framework) {
-			ginkgo.AfterEach(func(ctx context.Context) {
-				if f.ClientSet == nil {
-					// Test didn't reach f.BeforeEach, most
-					// likely because the test got
-					// skipped. Nothing to check...
-					return
-				}
-				e2enode.AllNodesReady(ctx, f.ClientSet, 7*time.Minute)
+			ginkgo.BeforeEach(func(ctx context.Context) {
+				ginkgo.DeferCleanup(func(ctx context.Context) {
+					gomega.Expect(e2enode.AllNodesReady(ctx, f.ClientSet, 7*time.Minute)).To(gomega.Succeed())
```

Comment:
this things make me nervous, please triple check we were asserting before and it got removed

Reply:
Thanks, I'll try to check it out. However, @pohly mentioned [here](https://github.com/kubernetes/kubernetes/pull/121139#issuecomment-1764665612) that he removed the error checking by accident.